pkg/controllers/readiness/wellknown_ready_controller.go

```diff
@@ -106,9 +111,16 @@ func (c *wellKnownReadyController) sync(ctx context.Context, controllerContext f
 	}
 
 	// the code below this point triggers status updates, unify status update handling in defer
-	statusUpdates := []v1helpers.UpdateStatusFunc{}
+	available := applyoperatorv1.OperatorCondition().
+		WithType("WellKnownAvailable").
+		WithStatus(operatorv1.ConditionFalse)
+	progressing := applyoperatorv1.OperatorCondition().
+		WithType(common.ControllerProgressingConditionName(controllerName)).
+		WithStatus(operatorv1.ConditionTrue)
```

Comment:
This condition will be changed when the the code enters line `138`. It might be changed from `progressing=false` to `progressing=true`, not ideal. Do we care ?

I'm just wondering if, in this case, it would be better to read the previously applied condition and use it.
Thoughts ?